scripts/create-clusters.sh

```diff
@@ -113,10 +113,16 @@ function create-clusters() {
     unset KUBECONFIG
   fi
 
-  # TODO(font): Configure insecure registry on kind host cluster. Remove once
-  # https://github.com/kubernetes-sigs/kind/issues/110 is resolved.
-  echo "Configuring insecure container registry on kind host cluster"
-  configure-insecure-registry-on-cluster 1
+  # Avoid configuring the insecure registry unless explicitly
+  # configured to do so. If the docker daemon is running on another
```

Comment:
Can we clarify this setup? For example, is kind being run inside a VM? If so, isn't this script running inside the VM as well? I ask because this logic configures an insecure registry for the cluster i.e. the docker daemon running inside of the control plane container.